contracts/script/IncredibleSquaringDeployer.s.sol

```diff
@@ -147,6 +147,7 @@ contract IncredibleSquaringDeployer is Script, Utils {
         erc20Mock = new ERC20Mock();
         priceAdapter = new PriceFeedAdapter();
         priceAdapter.addFeed("btc/usdc", address(0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43));
+        //priceAdapter.getPriceDia("BTC/USD");
```

Comment:
Should this be comments out? I think this is needed so when we start our local anvil node the priceAdapter is in the right state.